server/lib/pubsubNotifiers/notifiers/inMemory.js

```diff
@@ -0,0 +1,25 @@
+const PubSub = require('graphql-subscriptions').PubSub
+const { log } = require('../../util/logger')
+
+function InMemoryPubSub (config) {
+  const client = new PubSub()
+
+  this.publish = function publish ({ topic, compiledPayload }, context) {
+    let payload = compiledPayload(context)
+    // The InMemory pubsub implementation wants an object
+    // Whereas the postgres one would expect a string
+    try {
+      payload = JSON.parse(payload)
```

Comment:
It is missing `;`. Is not this project with a lint lib? Should this rule not be added there?

Reply:
All of this code is linted using [standard](https://standardjs.com/)